packages/flutter/lib/foundation.dart

```diff
@@ -43,6 +43,11 @@ export 'src/foundation/diagnostics.dart';
 export 'src/foundation/isolates.dart';
 export 'src/foundation/key.dart';
 export 'src/foundation/licenses.dart';
+export 'src/foundation/logging.dart' show
+  LoggingChannel,
+  debugEnableLogging,
+  debugLogEvent,
+  debugShouldLogEvent;
```

Comment:
I'm not sure if this is the desired way of not leaking internal details...  Happy to take another tack if preferred.

Reply:
What @jacob314 says. There's no such thing as a non-private internal.

Reply:
Got it.  I ended up on this road to avoid having a "manager" or "service" class (which felt like a bridge too far on the first pass) but leaning back in that direction now.

Thanks!